tests/unit/test_serialization.py

```diff
@@ -69,11 +70,21 @@ async def test_deserialization_fails(default_config_path, mock_registry_client):
     schema["name"] = "BadUser"
     schema["fields"][0]["type"] = "int"
     obj = {"name": 100, "favorite_number": 2, "favorite_color": "bar"}
-    writer = avro.io.DatumWriter(avro.io.schema.parse(json.dumps(schema)))
+    writer = avro.io.DatumWriter(avro.schema.parse(json.dumps(schema)))
     with io.BytesIO() as bio:
         enc = avro.io.BinaryEncoder(bio)
         bio.write(struct.pack(HEADER_FORMAT, START_BYTE, 1))
         writer.write(obj, enc)
         enc_bytes = bio.getvalue()
+    # Avro 1.11.0 does not assert anymore if the bytes io read function
```

Comment:
is this a left over comment? there is no `raises(AssertionError)`

Reply:
No, there is no more assertion errors. This comment server a purpose of explaining what is the behavior.
The test is for this line:
https://github.com/aiven/karapace/pull/333/files#diff-af2237d5ef00ea62612450b70e195a61f09b19b9aa0e224795942b2d3418a646R258